docs/configuration.rst

```diff
@@ -234,6 +238,22 @@ Config per Coin
 - MAXTOLEND, MAXPERCENTTOLEND, and MAXTOLENDRATE refer to their respective settings above, but are unique to the specified coin specifically.
 - There can be as many different coins as you want in coincfg, but each coin may only appear once.
 
+**Separate coin sections**
+
+This is an alternative layout for the coin config mentioned above. It provides the ability to change the minloansize per coin, but is otherwise identical in functionality.
+To use this configuration, make sure to comment out the line where coincfg is defined, then add a secion for each coin you wish to configure. These sections should come 
+at the end of the file, after the other options for the bot.
```

Comment:
Put this notice in bold so that people notice it at a glance, as it is very important.